forc/src/ops/forc_deploy.rs

```diff
@@ -68,7 +69,7 @@ pub async fn deploy(command: DeployCommand) -> Result<(), CliError> {
                         match client.submit(&tx).await {
                             Ok(logs) => {
                                 println!("Logs:\n{:?}", logs);
-                                Ok(())
+                                Ok(logs.into())
```

Comment:
Is this `.into()` converting the response of `.submit` into a contract id? If so, I think this is incorrect, since `submit` returns the id of the create tx, not of the underlying contract id contained in the ContractCreated output of the tx.

Reply:
That was the intention, yes. Oddly enough, all the tests pass? Why would that be then?

Reply:
This LGTM but I am hesitant to pull it in until we understand this question.

Reply:
Same...

Reply:
Fixed in 2a17d7f.